RadixWallet/Features/AssetsFeature/Components/HelperViews/ResourceBalance/ResourceBalanceView.swift

```diff
@@ -135,24 +135,25 @@ private extension ResourceBalance.ViewState.PoolUnit {
 struct ResourceBalanceView: View {
 	let viewState: ResourceBalance.ViewState
 	let appearance: Appearance
+	let hasBorder: Bool
```

Comment:
can `hasBorder` be true if `appearance` is `standard`? Otherwise it migth be an associated value of `case compact(hasBorder: Bool)`?

REALLY not important :D